controllers/package_controller_test.go

```diff
@@ -135,14 +129,10 @@ func TestReconcile(t *testing.T) {
 		sut := tf.newReconciler()
 		req := tf.mockRequest()
 		got, err := sut.Reconcile(ctx, req)
-		if err != nil {
-			t.Errorf("expected <nil> got <%s>", err)
-		}
+		assert.Nil(t, err)
```

Comment:
This works for sure, but for future reference. `assert.NoError(t, err)` may better communicate your intentions.